data/assets/educational/scale/scientist.asset

```diff
@@ -52,29 +54,30 @@ local updatePositionAction = {
   IsLocal = false
 }
 
-local resetPositionAction = {
-  Identifier = "os.os.scale_models.reset_scientist",
+local ResetPositionAction = {
+  Identifier = "os.scalemodels.ResetScientist",
   Name = "Reset Scientist position",
   Command = [[
-    openspace.globebrowsing.setNodePosition('Scale_Scientist', ']] .. earthAsset.Earth.Identifier .. [[', 34.201639, -118.171319)
-  ]],
-  Documentation = "Resets the Scientist back to it's actual location",
+    openspace.globebrowsing.setNodePosition('Scale_Scientist', ']] .. earthAsset.Earth.Identifier .. [[', ]] .. Location[1] .. "," .. Location[2] .. ")",
+  Documentation = "Resets the Scientist back to its actual location",
```

Comment:
its start location? :) The "actual location" of the scientists quickly becomes a bit philosophical